### PR TITLE
[PM-12400] Add private key regeneration SDK methods

### PR DESCRIPTION
**Pull request by Thomas-Avery**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-12400

## 📔 Objective

Add shared methods to enable private key regeneration in clients.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation
  team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: crates/bitwarden-wasm-internal/src/crypto.rs [hunk 1]
```diff
@@ -30,4 +33,15 @@ impl ClientCrypto {
     pub async fn initialize_org_crypto(&self, req: InitOrgCryptoRequest) -> Result<()> {
         Ok(self.0.crypto().initialize_org_crypto(req).await?)
     }
+
+    pub fn make_key_pair(&self) -> Result<MakeKeyPairResponse> {
```
Comment (dani-garcia) [inline crates/bitwarden-wasm-internal/src/crypto.rs:37]: We might want to add a couple of code comments to these functions, these are our public API with the clients.

**Review by dani-garcia — APPROVED** [review]
LGTM!

**Review by dani-garcia — APPROVED** [review]
LGTM!

**Review by quexten — APPROVED** [review]
Looks good to me, nice work!

**Comment by Hinton** [comment]
@Thomas-Avery This will need to be moved to https://github.com/bitwarden/sdk-internal. It should be as simple as changing remote and pushing the branch. All the history is preserved in the new repo.

**Comment by Thomas-Avery** [comment]
Closing, opened on sdk-internal here https://github.com/bitwarden/sdk-internal/pull/6

**Pull request closed without merging**